src/ui-kit/components/comments/comments.component.ts

```diff
@@ -204,4 +204,8 @@ export class SamCommentsComponent implements OnInit {
       0;
     return this.maxLength - textLength;
   }
+
+  commentAriaLabel(){
+    return "Enter your comments (" + this.charsRemaining() +  ")characters remaining)"
```

Comment:
This may be a better message for the textarea:
Enter your comments (Maximum characters allowed: this.maxLength)

Reply:
Updated text.